config/locales/en.yml

```diff
@@ -34,3 +34,4 @@ en:
   time:
     formats:
       military: "%H:%M:%S"
+      ampm: "%r"
```

Comment:
What's this do?

Reply:
This allows us to do `format: :ampm` in the `::I18n.localize` method. `"%r"` is a strftime shortcut for an am/pm version of HH:MM:SS (AM/PM).

Reply:
Here's all the strftime stuff available in Ruby, if you're curious: https://apidock.com/ruby/DateTime/strftime